drivers/serial/uart_nrf_sw_lpuart.c

```diff
@@ -907,7 +908,7 @@ static int lpuart_init(const struct device *dev)
 	const struct lpuart_config *cfg = get_dev_config(dev);
 	int err;
 
-	data->uart = device_get_binding(cfg->uart_name);
+	data->uart = DEVICE_DT_GET(DT_INST_BUS(0));
 	if (data->uart == NULL) {
```

Comment:
This will never be `NULL`. Call `device_is_ready()` instead.

Reply:
Fixed